example-mojaloop-backend/values.yaml

```diff
@@ -3,8 +3,15 @@
 # Declare global configurations
 global: {}
 
+nginx:
+  enabled: true
+
 ## Reference: https://github.com/bitnami/charts/blob/main/bitnami/kafka/values.yaml
```

Comment:
Here is an example Reference.

Reply:
@mdebarros same comment as above I have removed nginx from the backend